scss/_base_button.scss

```
@@ -9,17 +9,16 @@
   }
 
   %vf-button-dense-vertical-padding {
-    padding-bottom: calc(#{$spv-nudge - $sp-unit * 0.5} - 1px);
-    padding-top: calc(#{$spv-nudge - $sp-unit * 0.5} - 1px);
+    padding-bottom: calc(#{$spv-nudge - $sp-unit * 0.5} - #{$input-border-thickness});
```

I believe with new versions of sass we are using the sass support for `calc` allows to drop the `#{ }` interpolation of variables and use them directly inside calc.

I don't want to expand the scope of that to fix it everywhere, but at least the lines that we touched anyway could take benefit of that.
```suggestion
    padding-bottom: calc($spv-nudge - $sp-unit * 0.5 - $input-border-thickness);
```